src/vector/v.what.rast.label/v.what.rast.label.html

```diff
@@ -0,0 +1,37 @@
+<h2>DESCRIPTION</h2>
+
+<em>v.what.rast.label</em> retrieves raster values and labels from one
+or more given raster maps for each point stored in the input vector map.
+The name of the columns with the raster values and labels are
+respectively<i>ID_name-of-raster-map</i> and <i>name-of-raster-map</i>.
```

Comment:
Just out of curiosity: is the use of `-` (hyphen) allowed within column names in SQLite? My understanding was that this is problematic; v.rast.stats, for example, uses underscore by default.

Reply:
The text isn't clear. What I meant was that the column name will be "ID_" followed by the name of the raster layer from which the raster value is used. I'll change it.